docs/building-apps/wallet/stellar.mdx

```diff
@@ -479,6 +494,21 @@ suspend fun submitWithFeeIncrease() {
 }
 ```
 
+```typescript
+await stellar.submitWithFeeIncrease({
+  sourceAddress: kp,
+  timeout: 180,
+  baseFeeIncrease: 100,
+  operations: [
```

Comment:
currently no, there isn't as clean as a for adding a builder functions or similar in Typescript like there is in Kotlin

I'm not opposed to seeing if there's a better way for the user to add operations though, can you create a ticket to track?

Reply:
What if we accept a lambda (TransactionBuilder => TransactionBuilder) instead of operations list? 
Then I'd write something like
```ts
submitWithFeeIncrease(..., b => b.transfer(...))
```

Reply:
ya I agree I think that is nicer. I'll add in a next PR

Reply:
added here: https://github.com/stellar/typescript-wallet-sdk/pull/40

Reply:
Thanks @acharb , can you update this PR to reflect this changes? Then we can merge, LGTM!

Reply:
👍 https://github.com/stellar/stellar-docs/pull/212/commits/024c271a8057a7f1662b350dcc6d55a43c3722dc